naiserator.go

```diff
@@ -160,6 +164,23 @@ func (n *Naiserator) createOrUpdateMany(resources []runtime.Object) error {
 	return result.ErrorOrNil()
 }
 
+func (n *Naiserator) removeOrphanIngresses(logger *log.Entry, app *v1alpha1.Application) error {
+
+	if len(app.Spec.Ingresses) == 0 {
+		err := n.ClientSet.ExtensionsV1beta1().Ingresses(app.Namespace).Delete(app.Name, &v1.DeleteOptions{})
+
+		if errors.IsNotFound(err) {
+			return nil
+		}
+
+		logger.Infof("%s: Successfully deleted ingresses", app.Name)
```

Comment:
Kan kanskje være litt mer presis. Noe ala "Successfully deleted orphan ingresses for application: %s" ?

Reply:
Vi bør følge konvensjon, altså "%s: melding"

Reply:
lowercase på alle setningsledd.